### ISSUE TITLE
Connect directly to terminal without import kubeconfig

### ISSUE THREAD

**Issue by ndhieuops** [KubeOperator/webkubectl#28]
Hello, guys!
I want to know how to access the terminal without manually importing the kubeconfig file. 
The big idea is i already have some kubeconfig file in the local OS and the webkubectl will import these file and auto create some and my job is simply click on connect and access into it ! ( For example i have a lot of kubeconfig file and what if i import manually. It will take a lot of time)

**Comment by liqiang-fit2cloud** [comment]
> For example i have a lot of kubeconfig file and what if i import manually. It will take a lot of time

Ok...
I think batch upload will solve your problem.

**Comment by ndhieuops** [comment]
Yeah i got your point but it doesn't solved my problem because what if i have maybe 300 or 400 kubeconfig file (example). Yeah actually i can do it step by step and one by one but i just want to upgrade our job like everything will be automation. I have the directory and in this directory will have 400 kubeconfig file (example) i will create multi container and each container will manage some session (maybe 50 or 100 session). And each container will mount some kubeconfig file in some directory i defined so that i can access into the session without import manually

**Comment by ndhieuops** [comment]
@liqiang-fit2cloud @wan92hen did anyone have any recommendation please

**Comment by ndhieuops** [comment]
Okay !! Thank you very much
